xendit_pos/static/src/js/payment_xendit_pos.js

```diff
@@ -165,11 +177,13 @@ odoo.define('xendit_pos.payment', function (require) {
                 $("#invoice-link > a").text('Paid');
                 resolve(true);
             } else if(invoice.status == 'EXPIRED'){
-                
-                const paymentLine = this.pos.get_order().selected_paymentline;
                 $('#xendit-payment-status').text('Expired');
                 $("#invoice-link > a").text('Expired');
-                paymentLine.set_payment_status('retry');
+
+                const paymentLine = this.get_selected_payment();
+                if (paymentLine) {
+                    paymentLine.set_payment_status('retry');
```

Comment:
@andykim Can we keep all the payment statuses as an enum/constant (not sure what's the correct term) so it can be reused everywhere since I've seen it repeated several places

Reply:
@ikanyu Updated. we use `const` . `enum` can use for typescript